examples/cifar10/create_cifar10.sh

```diff
@@ -5,15 +5,26 @@ EXAMPLE=examples/cifar10
 DATA=data/cifar10
 DBTYPE=lmdb
 
+# Check if CAFFE_BIN is set
+if [ -z ${CAFFE_BIN+x} ]; 
+# if unset
```

Comment:
the `if unset` comment doesn't need its own line, and `then` can be on the same line:

`if [[ -z "$CAFFE_BIN" ]]; then`